### PR TITLE
Document classification data via HTTP headers

### PR DESCRIPTION
**Pull request by AnouckColson**
## Why

Platform.sh will soon send classification data to backends through HTTP headers.

## What's changed

Added a page to the security & compliance section with a dedicated table.

### CODE REVIEW

File: sites/platform/src/development/headers.md [hunk 1]
```diff
@@ -17,16 +17,26 @@ sidebarTitle: "Headers"
...
+- `X-Platform-Router`: The ID of the router that served the request. The router ID is the processor ID of the router container, specifically.
+- `X-Debug-Info`: This is a header added by the edge layer. It doesn't contain any sensitive information or anything that could be misused. It has no relation to the PHP debugging tool [Xdebug](https://xdebug.org).
+
+## Classification data headers
+
+Platform.sh sends classification data to your backend system through the following HTTP headers.
+
+| HTTP header          | Type    | Description                                                                                               |
+| -------------------- | ------- | --------------------------------------------------------------------------------------------------------- |
+| `Client-Cdn`         | string  | When a CDN that is supported by Platform.sh is used, this header displays its name (Fastly, Cloudflare, or Cloudfront). |
```
Comment (chadwcarlson) [inline sites/platform/src/development/headers.md:36]: s/Platform.sh/vendorName?

File: sites/platform/src/development/headers.md [hunk 2]
```diff
@@ -17,16 +17,26 @@ sidebarTitle: "Headers"
...
+- `X-Debug-Info`: This is a header added by the edge layer. It doesn't contain any sensitive information or anything that could be misused. It has no relation to the PHP debugging tool [Xdebug](https://xdebug.org).
+
+## Classification data headers
+
+Platform.sh sends classification data to your backend system through the following HTTP headers.
+
+| HTTP header          | Type    | Description                                                                                               |
+| -------------------- | ------- | --------------------------------------------------------------------------------------------------------- |
+| `Client-Cdn`         | string  | When a CDN that is supported by Platform.sh is used, this header displays its name (Fastly, Cloudflare, or Cloudfront). |     
+| `Client-Country`     | string  | The two-character ISO 3166-1 country code of the end-client IP (after CDN handling for the CDNs that Platform.sh supports).<BR/> The geolocation data sent through this header is provided by [MaxMind GeoLite2](https://dev.maxmind.com/geoip/geolite2-free-geolocation-data).|
```
Comment (chadwcarlson) [inline sites/platform/src/development/headers.md:37]: Same here?

File: sites/platform/src/security/security-threats/classification-data.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+description: Platform.sh allows you to send classification data to your backend system through HTTP headers to spot malicious IP addresses.
+weight: 3
+---
+
+To help spot malicious IP addresses, Platform.sh sends classification data to your backend system through the following HTTP headers.
+
+| HTTP header          | Type    | Description                                                                                               |
+| -------------------- | ------- | --------------------------------------------------------------------------------------------------------- |
+| `Client-Cdn`         | string  | When a CDN that is supported by Platform.sh is used, this header displays its name (Fastly, Cloudflare, or Cloudfront). |     
+| `Client-Country`     | string  | The two-character ISO 3166-1 country code of the end-client IP (after CDN handling for the CDNs that Platform.sh supports).<BR/> The geolocation data sent through this header is provided by [MaxMind GeoLite2](https://dev.maxmind.com/geoip/geolite2-free-geolocation-data).|
```
Comment (chadwcarlson) [inline sites/platform/src/security/security-threats/classification-data.md:12]: Same for these two lines

**Review by chadwcarlson — APPROVED** [review]
Aside from the `{{% vendor/name %}}` substitutions, all good to me 👍

**Review by flovntp — APPROVED** [review]

**Pull request merged**